easybuild/tools/modules.py

```diff
@@ -577,10 +591,15 @@ def mod_exists_via_show(mod_name):
 
         mods_exist = []
         for (mod_name, visible) in mod_names:
+            self.log.info("Checking whether %s exists...", mod_name)
             if visible:
                 mod_exists = mod_name in avail_mod_names
                 # module name may be partial, so also check via 'module show' as fallback
-                if not mod_exists and maybe_partial:
+                if mod_exists:
+                    self.log.info("Module %s exists (found in list of available modules)", mod_name)
+                elif not mod_exists and maybe_partial:
+                    self.log.info("Module %s not found in list of available modules, checking via 'module show'...",
```

Comment:
As you are using the (verbose) info method I find this (or the other) superflous as you will print `Checking whether %s exists based on output of 'module show` next (BTW: There is a `'` missing)

Reply:
@Flamefire The other log message is in another (sub)method, which could also be called from other places, so it's not superfluous imho. Missing single quote fixed in e483fe4